packages/cli/src/components/Inkuirer/InputQuestionProps.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
```

Comment:
This filename is confusing to me

Reply:
I think that was because of the refactor functionality